python_scripts requires update for 3.4 changes

```
Traceback (most recent call last):
  File "diffusion.py", line 166, in <module>
    tmgeom = gen_geom()
  File "diffusion.py", line 141, in gen_geom
    tetidxs[stored] = tidx
OverflowError: Python int too large to convert to C long
```
There should be some more down the line. And possibly the notebooks as well.

I created a small Flake8 plugin to do just that:
https://github.com/tristan0x/flake8-oist-steps

When run of STEPS_Example:
```
$ flake8 python_scripts 
python_scripts/diffusion/diffusion.py:139:13: E421 consider using steps.geom.UNKNOWN_TET constant instead of -1.
```

--- a/other_tutorials/OCNC2017/ex3_diff.py
+++ b/other_tutorials/OCNC2017/ex3_diff.py
@@ -15,6 +15,11 @@ import steps.solver as solvmod
 import steps.utilities.meshio as meshio 
 import steps.geom as stetmesh
 import steps.rng as srng
+try:
+    from steps.geom import UNKNOWN_TET
+except ImportError:
+    UNKNOWN_TET = -1
+
 
 ########################################################################
 
@@ -144,7 +149,7 @@ def gen_geom(meshfile):
 		
 		idx = mesh.findTetByPoint([xpnt, ypnt, zpnt])
 		
-		if (idx == -1): continue
+		if idx == UNKNOWN_TET: continue
 		if (idx not in tetidxs): 
 			tetidxs[filled] = idx
 			filled += 1

--- a/python_scripts/diffusion/diffusion.py
+++ b/python_scripts/diffusion/diffusion.py
@@ -38,6 +38,11 @@ import steps.model as smodel
 import steps.solver as solvmod
 import steps.geom as stetmesh
 import steps.rng as srng
+try:
+    from steps.geom import UNKNOWN_TET
+except ImportError:
+    UNKNOWN_TET = -1
+
 
 import steps.utilities.meshio as smeshio
 
@@ -136,7 +141,7 @@ def gen_geom():
         tidx = mesh.findTetByPoint([xcrd, ycrd, zcrd])
         
         # -1 was returned if point is outside the mesh:
-        if (tidx == -1): continue
+        if tidx == UNKNOWN_TET: continue
         if (tidx not in tetidxs):
             tetidxs[stored] = tidx
             stored += 1

